en/docs/insights/derive-insights.md

```diff
@@ -0,0 +1,364 @@
+| **API Name**          | The name of the API.                                                         |
+| **Application name:** | The name of the application. The name of the subscriber is also provided within brackets. |
+| **Usage Count**       | The request count.                                                           |
+
+#### Download pre-generated reports
+
+These widgets allow you to download monthly reports for the last three months.
+
+![Pre-generated reports](../assets/img/insights/configured-report.png){.cInlineImage-full}
+
+To download a pre-generated report, click **Download** for the relevant month and then click on the required format.
+
+### Alert Configuration
+
+This page allows you to configure alerts. These alerts allow you to proactively monitor your API ecosystem and take corrective measures for any abnormalities you find.
+
+You can configure alerts for each environment within your organization separately. You can configure alerts per API as well as modify/remove already-added alert configurations. Optionally, you can specify a list of emails for each alert configuration.
+
+Alerts are subject to a suppression policy to ensure you do not receive duplicate alert notifications. For each alert you configure, there is a 10-minute suppression window applied by default. The suppression interval is not configurable at present. In addition, note that the suppression policy is applicable per alert configuration.
+
+!!! info
+    - The maximum number of alerts that can be configured is 20 for each organization, environment, and tenant combination.
+    - Adding an email to an alert configuration is optional. The maximum number of emails per alert configuration is limited to 5.
+    
+The alert can be a latency alert or a traffic alert.
+
+#### Latency alerts
+
+Configure latency alerts to be notified if the response latency of APIs is greater than a predefined threshold. This is useful when you have APIs that should honor SLAs and when you want to know about slow APIs proactively.
+
+To configure a new latency alert, follow the steps below:
+
+1. In the Insights Portal, click **Alert Configuration** to open the **Alert Configuration** page.
+
+    The **Latency** tab is open by default.
+
+2. Verify that you are in the correct organization, and select the required environment. 
+
+3. In the **API Name** field, select the API for which you want to configure the alert.
+
+    !!! Info
+        Only the APIs that you have invoked at least once are listed here. For other APIs, you need to give the API name in the required format as instructed in the UI.
+        
+4. In the **Metric** field, select the required metric against which you want to evaluate the alert configuration.
+
+    !!! tip
+        The list includes all available options. If there are multiple metrics, you can select the required metric. If there is only one metric to select, that metric is selected by default, and the field is disabled. 
```

Comment:
This is not very clear. If there are multiple, can you still only select one? 
And you mean if there is only one metric in the list?

Reply:
Yes. If there are multiple metrics, you can select only one.  But if only one metric is available, that only available metric is already selected in this field and you can't edit the field.

Reply:
Do we need to specifically mention that? isn't that the expected behavior?

Reply:
If the documentation just instructs to select a value, the user might wonder why the field is greyed out and not editable. he/she might not realise it is because only one value is available.